README.md

```diff
@@ -104,6 +104,33 @@ export default function App() {
 </main>
 ```
 
+## Workarounds
```

Comment:
Not sure "workaround" is the right term here, since there isn't really a bug to work around. Maybe this section title can just be a catch-all for server-side rendering implementations? 
```suggestion
## Server-side Rendering
```